airbyte-integrations/connectors/source-jdbc/src/main/java/io/airbyte/integrations/source/jdbc/AbstractJdbcSource.java

```diff
@@ -426,6 +426,20 @@ public JdbcDatabase createDatabase(final JsonNode sourceConfig) throws SQLExcept
     return database;
   }
 
+  /**
+   * {@inheritDoc}
+   * @param database database instance
+   * @param catalog schema of the incoming messages.
+   * @throws SQLException
+   */
+  @Override
+  protected void logPreSyncDebugData(final JdbcDatabase database, final ConfiguredAirbyteCatalog catalog)
+      throws SQLException {
+    LOGGER.info("Data source product recognized as {}:{}",
```

Comment:
`.info` or `.debug`?

Reply:
This one should definitely be info. There will only be one log per sync, and we want it to be visible.